src/entitytypes.js

```diff
@@ -976,6 +976,35 @@ SlugIdArray.prototype.indexOf = function(slug) {
   return -1;
 };
 
+/** Determines whether it includes a certain element, returning true or false as appropriate. */
+SlugIdArray.prototype.includes = function(slug) {
+  return this.buffer.includes(slugIdToBuffer(slug));
+};
+
+/** The shift() method removes element. */
```

Comment:
which element?

Reply:
Good catch...the first element. Fixed it